### PR TITLE
Add Judoc comments for most of the functions

### PR DESCRIPTION
**Pull request by janmasrovira**

### CODE REVIEW

File: Stdlib/Data/Int/Ord.juvix [hunk 1]
```diff
@@ -4,42 +4,51 @@ open import Stdlib.Data.Int;
...
+== (ofNat m) (ofNat n) := m Nat.== n;
+== (negSuc m) (negSuc n) := m Nat.== n;
 == _ _ := false;
 
+--- Tests for inequality.
 infix 4 /=;
 /= : Int -> Int -> Bool;
 /= m n := not (m == n);
 
+--- Returns ;true; iff the first element is lesser or equal than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Int/Ord.juvix:22]:
```suggestion
--- Returns ;true; iff the first element is less than or equal than the second.
```

File: Stdlib/Data/Int/Ord.juvix [hunk 2]
```diff
@@ -4,42 +4,51 @@ open import Stdlib.Data.Int;
...
 infix 4 <=;
 builtin int-le
 <= : Int -> Int -> Bool;
 <= m n := nonNeg (n - m);
 
+--- Returns ;true; iff the first element is lesser than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Int/Ord.juvix:28]:
```suggestion
--- Returns ;true; iff the first element is less than the second.
```

File: Stdlib/Data/Int/Ord.juvix [hunk 3]
```diff
@@ -4,42 +4,51 @@ open import Stdlib.Data.Int;
...
 builtin int-lt
 < : Int -> Int -> Bool;
 < m n := m + 1 <= n;
 
+--- Returns ;true; iff the first element is greater than the second.
 infix 4 >;
 > : Int -> Int -> Bool;
 > m n := n < m;
 
+--- Returns ;true; iff the first element is greater or equal than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Int/Ord.juvix:39]:
```suggestion
--- Returns ;true; iff the first element is greater or equal to the second.
```

File: Stdlib/Data/List.juvix [hunk 4]
```diff
@@ -71,45 +88,56 @@ merge cmp (x :: xs) (y :: ys) :=
 merge _ nil ys := ys;
 merge _ xs nil := xs;
 
+--- 𝒪(𝓃). Returns a tuple for the first component has the items that
```
Comment (paulcadman) [inline Stdlib/Data/List.juvix:91]:
```suggestion
--- 𝒪(𝓃). Returns a tuple where the first component has the items that
```

File: Stdlib/Data/Nat/Ord.juvix [hunk 5]
```diff
@@ -12,35 +13,43 @@ builtin nat-eq
 == _ zero := false;
 == (suc n) (suc m) := n == m;
 
+--- Tests for inequality.
 infix 4 /=;
 /= : Nat → Nat → Bool;
 /= x y := not (x == y);
 
+--- Returns ;true; iff the first element is lesser or equal than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Nat/Ord.juvix:21]:
```suggestion
--- Returns ;true; iff the first element is less than or equal to the second.
```

File: Stdlib/Data/Nat/Ord.juvix [hunk 6]
```diff
@@ -12,35 +13,43 @@ builtin nat-eq
...
 infix 4 <=;
 builtin nat-le
 <= : Nat → Nat → Bool;
 <= zero _ := true;
 <= _ zero := false;
 <= (suc n) (suc m) := n <= m;
 
+--- Returns ;true; iff the first element is lesser than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Nat/Ord.juvix:29]:
```suggestion
--- Returns ;true; iff the first element is less than the second.
```

File: Stdlib/Data/Nat/Ord.juvix [hunk 7]
```diff
@@ -12,35 +13,43 @@ builtin nat-eq
...
 builtin nat-lt
 < : Nat → Nat → Bool;
 < n m := suc n <= m;
 
+--- Returns ;true; iff the first element is greater than the second.
 infix 4 >;
 > : Nat → Nat → Bool;
 > n m := m < n;
 
+--- Returns ;true; iff the first element is greater or equal than the second.
```
Comment (paulcadman) [inline Stdlib/Data/Nat/Ord.juvix:40]:
```suggestion
--- Returns ;true; iff the first element is greater than or equal to the second.
```

File: Stdlib/Data/Product.juvix [hunk 8]
```diff
@@ -41,5 +47,6 @@ second :
     → A × B';
 second f (a, b) := a, f b;
 
+--- Applies a function to both component.
```
Comment (paulcadman) [inline Stdlib/Data/Product.juvix:50]:
```suggestion
--- Applies a function to both components.
```

File: Stdlib/Function.juvix [hunk 9]
```diff
@@ -11,12 +12,15 @@ infixr 9 ∘;
...
 
+--- Always returns the first argument.
 const : {A : Type} → {B : Type} → A → B → A;
 const a _ := a;
 
+--- The identity function.
 id : {A : Type} → A → A;
 id a := a;
 
+--- Swaps the order of the argumets of the given function.
```
Comment (paulcadman) [inline Stdlib/Function.juvix:23]:
```suggestion
--- Swaps the order of the arguments of the given function.
```

**Review by paulcadman — COMMENTED** [review]
A nice improvement to the stdlib! I've just added suggestions to fix typos.

**Comment by janmasrovira** [comment]
thanks for all the corrections @paulcadman !

**Review by paulcadman — APPROVED** [review]

**Pull request merged**